### PR TITLE
Improve the help output

### PR DESCRIPTION
**Pull request by lu-zero**
- list the available templates
- enable colored output

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -4,36 +4,47 @@ mod toolchain;
...
-            _ => Err(()),
+impl Templates {
+    pub fn info() -> String {
+        let mut info = "Available built-in templates:\n".to_string();
+        for (names, description) in Templates::descriptions() {
+            std::fmt::write(
+                &mut info,
+                format_args!("{:<15} {}\n", names[0], description[0]),
+            )
+            .unwrap();
```
Comment (Luni-4) [inline src/lib.rs:44]: Do not use `unwrap` inside the lib, let's return a `Result`

**Review by Luni-4 — CHANGES_REQUESTED** [review]
Just a simple nit

File: src/lib.rs (hunk 1 above)
Comment (lu-zero) [reply src/lib.rs:44]: It cannot fail.

Comment (Luni-4) [reply src/lib.rs:44]: Yeah, but the use of an `unwrap` could be contraindicated in a lib in my opinion, anyway is just a nit

**Pull request merged**